src/realmGeol.ttl

```diff
@@ -57,307 +288,1624 @@ sorelsp:hasUpperBoundary rdf:type owl:ObjectProperty .
+soreag:Member rdf:type owl:Class ;
+              rdfs:subClassOf soreag:LithostratigraphicUnit ;
+              dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+              dcterms:source <http://www.stratigraphy.org/index.php/ics-stratigraphicguide> ;
+              skos:definition "The formal lithostratigraphic unit next in rank below a formation. (ICS, Online, December 2019)"@en ;
+              skos:prefLabel "member"@en .
 
 
 ###  http://sweetontology.net/realmGeol/Mesosphere
 soreag:Mesosphere rdf:type owl:Class ;
-                owl:equivalentClass soreag:UpperMantleRigid ;
-                rdfs:label "mesosphere"@en .
+                  owl:equivalentClass soreag:UpperMantleRigid ;
+                  rdfs:subClassOf soreag:UpperMantle ;
+                  rdfs:label "mesosphere"@en .
+
+
+###  http://sweetontology.net/realmGeol/MineralProvince
+soreag:MineralProvince rdf:type owl:Class ;
+                       rdfs:subClassOf soreag:GeologicProvince ;
+                       dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                       dcterms:source <http://www.glossaryofgeology.org> ;
+                       skos:altLabel "Metallogenic province"@en ;
+                       skos:definition "An area characterized by a particular assemblage of mineral deposits, or by one or more characteristic types of mineralization. A metallogenic province may have had more than one episode of mineralization.  (Neuendorf et al., 2011)"@en ;
+                       skos:prefLabel "mineral province"@en .
 
 
 ###  http://sweetontology.net/realmGeol/Moho
 soreag:Moho rdf:type owl:Class ;
-          owl:equivalentClass soreag:Mohorovi_i_Discontinuity ;
-          rdfs:subClassOf soreag:GeologicBoundary ;
-          rdfs:label "moho"@en .
+            owl:equivalentClass soreag:Mohorovi_i_Discontinuity ;
+            rdfs:subClassOf soreag:GeologicBoundary ;
+            rdfs:label "moho"@en .
 
 
 ###  http://sweetontology.net/realmGeol/Mohorovi_i_Discontinuity
 soreag:Mohorovi_i_Discontinuity rdf:type owl:Class ;
-                              rdfs:label "mohorovicic discontinuity"@en .
+                                rdfs:subClassOf soreag:GeologicBoundary ;
+                                rdfs:label "mohorovicic discontinuity"@en .
+
+
+###  http://sweetontology.net/realmGeol/Monocline
+soreag:Monocline rdf:type owl:Class ;
+                 rdfs:subClassOf soreag:StructuralFeature ;
+                 dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                 dcterms:source <https://doi.org/10.1093/acref/9780199653065.001.0001> ;
+                 skos:definition "A one-limbed flexure on either side of which the strata are horizontal or dip uniformly at low angles. (Allaby, 2013)"@en ;
+                 skos:prefLabel "monocline"@en .
+
+
+###  http://sweetontology.net/realmGeol/MyloniteZone
+soreag:MyloniteZone rdf:type owl:Class ;
+                    rdfs:subClassOf soreag:StructuralFeature ;
```

Comment:
shouldn't this be a subclass of shear zone?

Reply:
Do we start to develop subclasses of shear zones @geoderekh? We have the end members of brittle shear and ductile shear and what else in between?

Reply:
If you have different kinds of shear-zone in your data, then OK. But don't make this a stamp-collecting exercise ...

Reply:
My vote to avoid stamp collecting would be to deprecate 'mylonite zone' and avoid the problem.

Reply:
Agreed

Reply:
We could make a whole ontology about the spectrum existing between faults/brittle SZs and ductile SZs, but I dont think that is in scope here. Having two end members with a bunch of altLabels where relevant.